### ISSUE TITLE
Support for S3 Storage Class "Standard - IA"

### ISSUE THREAD

**Issue by pableu** [aws/aws-cli#1508]
I would like to upload a file to S3 and set it the the new "standard - infrequent access" storage class (see [here](https://aws.amazon.com/blogs/aws/aws-storage-update-new-lower-cost-s3-storage-option-glacier-price-reduction/)).

I couldn't figure out how to do this, and I think it's simply not implemented or at least not well documented. 

    $ aws --version
    aws-cli/1.8.5 Python/2.7.9 Linux/3.16.0-4-amd64

And the help isn't very helpful :-)

    $ aws s3 cp help
       ...
       --storage-class  (string)  The  type  of storage to use for the object.
       Valid choices are: STANDARD | REDUCED_REDUNDANCY.  Defaults  to  'STANDARD'

**Comment by tombrossman** [comment]
Came here to file this exact same issue. Their [storage classes documentation](http://docs.aws.amazon.com/AmazonS3/latest/dev/storage-class-intro.html) says about Glacier *"You cannot specify GLACIER as the storage class at the time that you create an object. You create GLACIER objects by first uploading objects using STANDARD, RRS, or STANDARD_IA as the storage class. Then, you transition these objects to the GLACIER storage class using lifecycle management."* and I suspect for now that we have to set a very short lifecycle (0 days, 1 day?) as a workaround.

Trying to use the new value anyway returns an 'Invalid choice' error:
```
aws s3 sync /PATH/SOURCE s3://PATH/BUCKET --storage-class standard_ia
usage: aws [options] <command> <subcommand> [parameters]
aws: error: argument --storage-class: Invalid choice, valid choices are:

STANDARD                                 | REDUCED_REDUNDANCY
```

**Comment by jamesls** [comment]
This was only in the lower level `s3api` commands.  It's now been added to the high level S3 commands via https://github.com/aws/aws-cli/pull/1511, so you'll be able to use STANDARD_IA with the `aws s3 cp` commands.

**Comment by jamesls** [comment]
Now available in 1.8.6 of the CLI.
